integration-libs/opf/components/opf-checkout-payments/opf-checkout-payments.component.html

```diff
@@ -0,0 +1,26 @@
+<ng-container *ngIf="activeConfiguratons$ | async as activeConfiguratons">
+  <div>
+    <div class="title">{{ 'opf.checkout.paymentOption' | cxTranslate }}</div>
```

Comment:
consider to use `h3` element to improve a11y

Reply:
Fixed in other PR extending logic of this one.
https://github.com/SAP/spartacus/pull/16802